packages/bus-merge-binding-factory/lib/ActorMergeBindingFactory.ts

```diff
@@ -0,0 +1,45 @@
+import type { IAction, IActorArgs, IActorOutput, IActorTest, Mediate } from '@comunica/core';
+import { Actor } from '@comunica/core';
+import type { IActionContext } from '@comunica/types';
+
+/**
+ * A comunica actor for the creation of merge handlers for binding context keys.
+ *
+ * Actor types:
+ * * Input:  IActionMergeBindingFactory:      TODO: fill in.
+ * * Test:   <none>
+ * * Output: IActorMergeBindingFactoryOutput: TODO: fill in.
+ *
+ * @see IActionMergeBindingFactory
+ * @see IActorMergeBindingFactoryOutput
+ */
+
+export abstract class ActorMergeBindingFactory extends Actor<IActionMergeBindingFactory,
+IActorTest,
+IActorMergeBindingFactoryOutput> {
+  /**
+  * @param args - @defaultNested {<default_bus> a <cc:components/Bus.jsonld#Bus>} bus
+  */
+  public constructor(args: IActorMergeBindingFactoryArgs) {
+    super(args);
+  }
+}
+
+export interface IActionMergeBindingFactory extends IAction {
+  context: IActionContext;
```

Comment:
Not sure if that would be useful at this stage, but we could also consider passing the whole `Bindings` object here. WDYT?

Reply:
I don't know how that would work or the benefit. Would you then call the merge-bindings-context mediator each time you need to merge bindings?